generator/templates/structfield.gotmpl

```diff
@@ -3,27 +3,9 @@
     // {{ template "docstring" . }}
     {{- template "propertyValidationDocString" .}}
   {{- end}}
-{{ pascalize .Name}} {{ template "schemaType" . }} `
-  {{- $prop := . }}
-  {{- range $i, $structTag := $.StructTags }}
-    {{- if gt $i 0 }} {{ end }}
-    {{- template "structTag" dict "Tag" $structTag "Schema" $prop }}
-  {{- else }}
-    {{- template "structTag" dict "Tag" "json" "Schema" $prop }}
-  {{- end }}
-  {{- if .XMLName }} xml:"{{ .XMLName }}"{{ end }}
-  {{- if .CustomTag }} {{ .CustomTag }}{{ end }}`
+{{ pascalize .Name}} {{ template "schemaType" . }} {{ $.PrintTags }}
```

Comment:
question here: why do we need to call PrintTags from the parent? (i.e. `$.PrintTags`)

Reply:
It was just me trying to fix the test, that didn't work.